internal/namespaces/instance/v1/custom_security_group.go

```diff
@@ -324,6 +330,36 @@ func securityGroupUpdateCommand() *core.Command {
 				Name: "organization-default",
 			},
 		},
+		Examples: []*core.Example{
+			{
+				Short:   "Change the name of the given security group",
+				Request: `{"security_group_id": "da1550ea-af6c-4441-9ddc-7565c5e8a7dc", "name": "foobar"}`,
+			},
+			{
+				Short:   "Change the description of the given security group",
+				Request: `{"security_group_id": "69e03584-613b-49b9-87ec-a57d540a91d6", "description": "foobar"}`,
+			},
+			{
```

Comment:
Do we need both `Enable stateful security group` and `Disable stateful security group`?